src/plot_cartogram.R

```diff
@@ -282,3 +286,256 @@ rm_facet_clip <- function(svg_in, file_out, width){
+    guides(fill = guide_colorsteps(
+      title = "",
+      nrow = 1,
+      direction = 'horizontal',
+      label.position = "bottom",
+      barwidth = 12,
+      barheight = 0.6,
+      background = element_rect(fill = NA),
+      show.limits = TRUE,
+      even.steps = FALSE
+    )) +
+    theme(legend.background = element_rect(fill = NA),
+          text = element_text(family = font_legend, color = text_color, size = 6.5))
+}
+
+
+
+# national level flow time series  - instagram versioning (slide 1)
+#' @description Compose the final plot and annotate
+#' @param file_png Filepath to save to
+#' @param plot_left The national plot to position on the left
+#' @param date_start First day of focal month
+#' @param width Desired width of output plot
+#' @param height Desired height of output plot
+#' @param color_bknd Plot background color
+#' @param text_color Color of text in plot
+#' @param flow_label Flow percentile label placed above legend
+#' @param source_label Source label placed in bottom right of plot
+#' @param restyle_legend Restylizing legend national flow timeseries plot
+#' @param enable_showtext Enabling fonts 
+national_ig <- function(file_png, plot_nat, date_start, width, height, color_bknd,
+                        text_color, flow_label, source_label, restyle_legend, enable_showtext){
+  plot_month <- lubridate::month(date_start, label = TRUE, abbr = FALSE)
+  plot_year <- lubridate::year(date_start)
+  
+  # import fonts
+  font_legend <- 'Noto Sans Mono'
+  font_add_google(font_legend)
+  showtext_opts(dpi = 300, regular.wt = 200, bold.wt = 700)
+  showtext_auto(enable = enable_showtext)
+    
+  # usgs logo
+  usgs_logo <- magick::image_read('in/usgs_logo.png') %>%
+    magick::image_colorize(100, text_color) |> magick::image_scale('250x')
+  
+  # streamflow title
+  title_flow <- magick::image_read('in/streamflow.png')
+  
+  plot_margin <- 0.025
+  
+  # background
+  canvas <- grid::rectGrob(
+    x = 0, y = 0, 
+    width = 16, height = 9,
+    gp = grid::gpar(fill = color_bknd, alpha = 1, col = color_bknd)
+  )
+  
+  # Extract from plot
+  plot_legend <- get_legend(restyle_legend)
+  
+  # compose final plot
+  ggdraw(ylim = c(0,1), 
+         xlim = c(0,1)) +
+    # a white background
+    draw_grob(canvas,
+              x = 0, y = 1,
+              height = 0.37, width = 0.37,
+              hjust = 0, vjust = 1) +
+    # national-level plot
+    draw_plot(restyle_legend+theme(legend.position = 'none',
+                              axis.title.x.bottom = element_text(size = 14),
+                              axis.title.x.top = element_text(size = 14),
+                              axis.text.x.bottom = element_text(size = 6),
+                              axis.text.y = element_text(size = 6)),
+              x = plot_margin*-10,
+              y = 0.29,
+              height = 0.50 ,
+              width = plot_margin*60) +
+    # add legend
+    draw_plot(plot_legend,
+              x = plot_margin*20,
```

Comment:
consider using relative positioning rather than a high multiplier of the `plot_margin` to position this. If `plot_margin` were changed to make more space around the plot, this would move other things around a lot